src/serialization/serialization.h

```diff
@@ -173,6 +202,12 @@ inline bool do_serialize(Archive &ar, bool &v)
     if (!r || !ar.good()) return false;			\
   } while(0);
 
+/*! \macro FIELD_F(f)
+ *
+ * \brief tags the field with the variable name and then serializes it (for use in free function)
+ */
+#define FIELD_F(f) FIELD_N(#f, v.f)
```

Comment:
I don't understand what this does, other than it allows the macro to be used outside of a class scope. Presumably that is the goal here?

Reply:
Yes, that's the current design goal with the Seraphis lib AFAIK: separate class definitions and their serialization.